apps/bridge-dapp/src/hooks/useCurrentTransaction.ts

```diff
@@ -0,0 +1,19 @@
+import { useMemo } from 'react';
+import { useLocation } from 'react-router';
+import { BRIDGE_TABS } from '../constants';
+
+/**
+ * Returns the current transaction tab on the bridge
```

Comment:
Would a better name for this be useCurrentTransactionTab? Seems like a path thing, current name almost indicates it has something to do with a literal transaction. It's identical to `useCurrentTx` semantically.